curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/616965351e74d4689eb6de30.md

```diff
@@ -7,6 +7,8 @@ dashedName: step-5
 
 # --description--
 
+You can have multiple self-closing `meta` elements on a web page. What distinguishes one `meta` element from the other is the attribute. Every `meta` element with a different attribute must also stand alone.
```

Comment:
I would word this: "Each `meta` element that has a different attribute must stand alone."

But I'm not 100% sure what "stand alone" means. Perhaps you should say "You should add a new `meta` element for each attribute you want to specify" ?

Reply:
Are you saying I should replace all the new texts I added with that?

Reply:
No, the other text is fine. I'm just saying "must also stand alone" isn't sufficiently clear to me. Is there another way you can word this?

Reply:
I think it's better to have it like you said: "You should add a new meta element for each attribute you want to specify".